ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- **Read (4 min) “[About OpenSearch](https://opensearch.org/about.html)”** to understand the primary project components and development principles
+- Optional - Read founding documents to understand the project history
+  - Read (8 min) “[Introducing OpenSearch](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)” from 2021-04-12
+  - Read (6 min) “[Stepping up for a truly open source Elasticsearch](https://aws.amazon.com/blogs/opensource/stepping-up-for-a-truly-open-source-elasticsearch/)” from 2021-01-21
+
+### How we work
+
+We model our way of working on “[the Apache Way](https://apache.org/theapacheway)” (optional - read 4 min). Specifically, we’ve borrowed and adapted:
+
+- *Community Over Code:* A healthy community is a higher priority than good code. Strong communities can always rectify problems with their code, whereas an unhealthy community will likely struggle to maintain a codebase in a sustainable manner.
+- *Earned Authority:* All individuals are given the opportunity to participate, but their influence is based on publicly earned merit – what they contribute to the community. Merit lies with the individual, does not expire, is not influenced by employment status or employer, and is non-transferable (merit earned in one project cannot be applied to another).
+- *Open Communications:* All communications related to code and decision-making should be publicly accessible to ensure asynchronous collaboration, as necessitated by a globally-distributed community.
+- *Community of Peers:* Individuals participate, not organizations. Roles are equal irrespective of title, votes hold equal weight, and contributions are made on a volunteer basis (even if paid to work on code).
+- *Progress over Perfection:* Struggling with a test failure? Can’t quite figure out why that data structure isn’t doing what you expect? Feeling defeated by the complexities of that massively distributed compute engine? The community idea exists to help each other through difficult problems. So fail early, commit often, ask for help, and don’t be afraid.
+
+### Contributor expectations
+
+Contributors are subject to a code of conduct, and it’s essential that all contributions are original work.
+
+- **Read (6 min)** [the Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct)
+- **Read (6 min)** [the OpenSearch CONTRIBUTING guide](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md)
+  - Please make sure you understand the implications of the [Developer Certificate of Origin](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin)
+
+### Resources and links
+
+There are more than 90 [GitHub repositories](https://github.com/opensearch-project/OpenSearch) that are part of the OpenSearch project. Explore the list to get a sense of the project breadth and scope.
```

Comment:
Should this be more than 100?

Reply:
This is only counting public repos (it excludes archived repos), which is currently 97. But the link is wrong. I'll update to https://github.com/orgs/opensearch-project/repositories?q=&type=public&language=&sort=